frontend/src/pages/about/index.js

```diff
@@ -7,17 +7,20 @@ import Strategy from "../../components/page-about/index/strategy"
 import Belief from "../../components/page-about/index/belief"
 import LeadershipSection from "../../components/page-about/index/leadership";
 import Feedback from "../../components/page-about/index/feedback"
+import Story from "../../components/page-about/index/story"
 
 export const Head = () => <Seo title="About" />;
 
 const AboutPage = () => (
   <Layout>
     <div className="max-w-container">
       <div>Our Story section</div>
+      <div>Our Beliefs section</div>
```

Comment:
Don't put this line in, this is just a placeholder, rather put the `<Belief />` where it's supposed to go